app/src/main/java/com/example/polina/socialnetwork/IntroActivity.java

```diff
@@ -25,12 +26,15 @@ protected void onCreate(Bundle savedInstanceState) {
     public void onClick(View v){
         switch (v.getId()){
             case R.id.sign_up:
+
                intent = new Intent(this, SignUpActivity_.class);
                 break;
             case R.id.sing_in:
+
                 intent = new Intent(this, SignInActivity_.class);
                 break;
         }
+        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
```

Comment:
what for? after click on back button on Sign In/Up screens, user should back to Intro screen